spacy/language.py

```diff
@@ -1123,6 +1164,7 @@ def evaluate(
             arguments for specific components.
         scorer_cfg (dict): An optional dictionary with extra keyword arguments
             for the scorer.
+        exclude (Iterable[str]): Names of components that shouldn't be evaluated.
```

Comment:
Hm, but if we're not running the component, we're not just not evaluating it. The rest of the pipeline also can't use its predictions, which could cause problems.

I think we should probably drop this option, and make people use `{disable, select}_pipes` for evaluate.

Reply:
Sure, that works, too. I just wanted to make it consistent. Because you might have a component that you want to train and not evaluate, for whatever reason. But maybe that's confusing.